harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -601,19 +611,39 @@ def make_workloads() -> workload.Stream:
         )
         controller.run()
 
-    @pytest.mark.skipif(not HAVE_APEX or not torch.cuda.is_available(), reason="no gpu available")
+    @pytest.mark.skipif(not torch.cuda.is_available(), reason="no gpu available")
     @pytest.mark.gpu
-    def test_apex_amp(self) -> None:
-        apex.amp.register_float_function(torch, "sigmoid")
-
-        def make_workloads() -> workload.Stream:
-            trainer = utils.TrainAndValidate()
-            yield from trainer.send(steps=1, validation_freq=1, scheduling_unit=1)
+    @pytest.mark.parametrize(
+        "trial_class,assert_output_float16",
+        [
+            (pytorch_onevar_model.OneVarApexAMPTrial, False),
+            (pytorch_onevar_model.OneVarAutoAMPTrial, False),
+            (pytorch_onevar_model.OneVarManualAMPTrial, True),
```

Comment:
since the only case that produces `assert_output_float16` seems to be the manual case, it seems like that assertion isn't very meaningful.  You could simplify the test by removing that assertion with no loss of test coverage on the product I think.

Reply:
Fine with me! Removed in 7e6ac53.